CommunityService_Fonts_Colours_v1.html

```diff
@@ -62,27 +61,52 @@ <h1>Community Service Page</h1>
 
 
 
-<br>Students will spend a day completing community service in a small group with friends in the Tawa basin, supervised by a staff member. This is a highly valuable experience where students have the chance to interact with the elderly, helping them with gardening, painting, etc., or assisting local food banks, kindergartens, hospice shops, and charities for the day. Students will usually walk to their assigned job with a staff member; however, a very small number of jobs will be on the fringes of the Tawa Basin, and therefore they may be driven by a staff member in their groups.<br></p>
+<br>Students will spend a day completing community service in a small group with friends in the Tawa basin, supervised by a staff member. This is a highly valuable experience where students have the chance to interact with the elderly, helping them with gardening, painting, etc., or assisting local food banks, kindergartens, hospice shops, and charities for the day. Students will usually walk to their assigned job with a staff member; however, a very small number of jobs will be on the fringes of the Tawa Basin, and therefore they may be driven by a staff member in their groups.</p>
 			
 			<center>
 			<img src="images/iMAGE2.jpg" alt="iMages">
 			<img src="images/iMAGE2.jpg" alt="iMages">
 			</center>
 				
-		</div>	
-								</div>	
+					</div>	
 			  <button onclick="topFunction()" id="myBtn" title="Go to top">TOP</button>
-
 			
 		</div>
-			<div class="footer">copyright &copy; Lynne Farah 2024, all rights reserved.</div>
-		</div>
+		
+		<div class="box3">	

```

Comment:
Instead of ending my webiste with a footer I added a box above the footer called box 3 to add quick links and the contact information of the secretary.